UpdateVisibleServicesCache/function.json

```diff
@@ -0,0 +1,38 @@
+{
+  "disabled": false,
+  "bindings": [
```

Comment:
since we're rewriting this logic, it's worth consider refacator the whole thing using cosmosdb change feed to write into table storage in order to 1) have repeateable processable data streams  2) reduce the probability that the cached data gets out of sync (currently this is not an unlikely event since we're writing into a shared json blob using a lock) 3) get rid of the scheduled task here and have the cache update in real time

Reply:
Thanks @gunzip your is a good point. At the moment is not in scope to rewrite this logic. The objective of this PR is to decouple the orchestrator from the `fn-admin` to avoid drop of performance in `fn-services` API and `devportal`. We know the problem and we'll refactor this system later.